.github/workflows/ci.yml

```diff
@@ -88,32 +70,13 @@ jobs:
 
       - run: echo "::add-mask::${{ secrets.LICENSE_SERVER_URL }}"
 
-      - name: Setup .NET SDK (v6.0)
-        uses: actions/setup-dotnet@v1
-        with:
-          dotnet-version: '6.0.x'
-
-      # Unity is about 4GB in size and takes 20 minutes to download and install.
-      # Caching still takes some time, but brings it down to about 5 minutes.
-      - name: Restore cached Unity installation
-        id: cache-unity
-        uses: actions/cache@v2
-        with:
-          path: ${{ matrix.unity-installation-path }}
-          key: ${{ matrix.os }}-${{ matrix.unity-version }}-${{ matrix.unity-version-changeset }}
-
-      - name: Restore previous build
-        uses: actions/cache@v2
-        with:
-          path: samples/artifacts/builds
-          # using hash of package.json to bust cache on release builds which modify it
-          key: samples/artifacts/builds|${{ matrix.os }}-${{ matrix.unity-version }}-${{ matrix.unity-version-changeset }}-${{ hashFiles('package/package.json') }}
```

Comment:
This was caching the android (smoke-test) player build but that doesn't make sense (with the key based only on the package.json) - we actually weren't testing code the changes made in the PRs if the cache key hit...